entrypoint.py

```diff
@@ -0,0 +1,24 @@
+#!/bin/python3
+
+import os
+import argparse
+import re
+import subprocess
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+                    prog='ProgramName',
+                    description='What the program does',
+                    epilog='Text at the bottom of help')
+    parser.add_argument('--file_regex')
```

Comment:
```suggestion
    parser.add_argument('--file-regex')
```
and then you can access the variable with `args.file_regex`.

Reply:
Didn't know that argparse replaces `-` with `_` in the variable name

Reply:
did it for all occurrences